app/src/main/java/kr/co/softcampus/sopt_assignment1/ui/fragment/Onboarding3Fragment.kt

```diff
@@ -0,0 +1,34 @@
+package kr.co.softcampus.sopt_assignment1.ui.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import kr.co.softcampus.sopt_assignment1.R
+import kr.co.softcampus.sopt_assignment1.databinding.FragmentOnboarding3Binding
+
+class Onboarding3Fragment : Fragment() {
+
+    private var _binding: FragmentOnboarding3Binding? = null
+    private val binding get() = _binding ?: error("Binding 이 초기화 되지 않았습니다.")
+
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
```

Comment:
```suggestion
    ): View {
```
binding.root의 경우 nullable하지않은 View 타입의 객체를 반환하기 때문에 이렇게만 쓰셔도 괜찮습니다!

Reply:
넵~!